src/HPRT/templates/screenings.html

```diff
@@ -157,21 +184,27 @@ <h1 style ="margin-right: 5px;">Screening Instruments</h1>
         <div id="collapseSix" class="collapse" data-parent="#accordion">
             <div class="card-body">
                 <a href = "{%url 'GH_new' patient %}" class="btn btn-primary border">New General Health/Physical Functioning</a>
-                <table class="table table-bordered table-striped">
-                    <thead>
-                        <tr>
-                            <th>Date</th>
-                            <th>Doctor</th>
-                            <th>Score</th>
-                            <th></th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {% for q in gh_list %}
-                            <tr><td>{{q.date}}</td> <td> {{q.doctor}}</td> <td>{{q.score}}</td><td><a href = "{%url 'GH_pdf' q.pk %}"> Download PDF</a></td></tr>
-                        {% endfor %}
-                    </tbody>
-                </table> 
+                {% if gh_list %}
+                    <table class="table table-bordered table-striped">
+                        <thead>
+                            <tr>
+                                <th>Date</th>
+                                <th>Doctor</th>
+                                <th>Score</th>
+                                <th></th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for q in gh_list %}
+                                <tr><td>{{q.date}}</td> <td> {{q.doctor}}</td> <td>{{q.score}}</td><td><a href = "{%url 'GH_pdf' q.pk %}"> Download PDF</a></td></tr>
+                            {% endfor %}
+                        </tbody>
+                    </table>
+                {% else %}
+                <h5>No current records.</h5>
+                {% endif %}
+                <div id="GH"></div>
+                <script>makePlot("GH", {{GH_x|safe}}, {{GH_y|safe}})</script>
```

Comment:
I can give more information about the params in the function definition. The safe keyword makes it so that the inputs from Django are not preprocessed. If I don't include it, the graph will not render. I'll add a comment about that.